### DISCUSSION TITLE
Cannot read properties of null (reading 'addEventListener')

### DISCUSSION THREAD

**Discussion by Kolomsim** [requarks/wiki#5874] [Help / Questions]
Hello everyone. I tried to add a script to click a button /some tag for a slider or other content, but it gives the error "Cannot read properties of null (reading 'addEventListener')". I understand that the error may be caused by the fact that the element has not yet been initialized on the page, but the script is located before </body> + added DOMContentLoaded for safety, but even so the error does not disappear. I just started learning JS so I may not know everything.

**Comment by JuL314** [comment]
Hi !
Same problem here, I found this :

https://github.com/requarks/wiki/discussions/5573#discussioncomment-3522232

But it still does not work for me, maybe you'll be luckier ! Let me know if it works.

**Comment by Kolomsim** [comment]
Yes, just yesterday I found an article about this event, but it is not entirely clear what needs to be put instead of "vue". Any ideas?

**Comment by Tuege** [comment]
@JuL314 @Kolomsim 
If you look at NGPixel's last reply that actually works #5573
